src/components/form/components/select.js

```diff
@@ -31,6 +32,7 @@ const Select = ({
       className={classnames('select', className, {
         [`is-${calculatedSize}`]: calculatedSize,
         [`is-${color}`]: color,
+        'is-expanded': fullwidth,
```

Comment:
Looking at this example from Bulma's documentation (https://bulma.io/documentation/form/general/#form-addons):
```html
<div class="field has-addons">
  <div class="control is-expanded">
    <div class="select is-fullwidth">
      <select name="country">
        <option value="Argentina">Argentina</option>
        <option value="Bolivia">Bolivia</option>
        <option value="Brazil">Brazil</option>
        <option value="Chile">Chile</option>
        <option value="Colombia">Colombia</option>
        <option value="Ecuador">Ecuador</option>
        <option value="Guyana">Guyana</option>
        <option value="Paraguay">Paraguay</option>
        <option value="Peru">Peru</option>
        <option value="Suriname">Suriname</option>
        <option value="Uruguay">Uruguay</option>
        <option value="Venezuela">Venezuela</option>
      </select>
    </div>
  </div>
  <div class="control">
    <button type="submit" class="button is-primary">Choose</button>
  </div>
</div>
```

It appears that the class of the surrounding `control` should be `is-expanded`, but the class of the `select` should be `is-fullwidth`.

Reply:
Ah yes, my mistake. I just copy pasted from the controller.  Updated.